components/console-backend-service/README.md

```diff
@@ -13,17 +13,17 @@ See the [GraphQL schema definition](internal/gqlschema/schema.graphql) file for
 
 Use the following tools to set up the project:
 
-* [Go](https://golang.org)
-* [Docker](https://www.docker.com/)
+- [Go](https://golang.org)
+- [Docker](https://www.docker.com/)
 
 ## Usage
 
 ### Run a local version
 
-To run the application without building the binary, run this command:
+To run the application without building the binary, set your `$KUBECONFIG` environment variable, then execute:
```

Comment:
```suggestion
To start the application without building the binary, set your `$KUBECONFIG` environment variable, and run:
```

Reply:
or "To run the application... , and use this command:" but I prefer the 1st version. I changed it this way because of the [recommendation in guidelines](https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-terminology) and didn't want to use the verb `run` twice.